README.md

```diff
@@ -168,6 +186,10 @@ Below are some commonly encountered issues and possible ways to resolve them. If
 - The app gets stuck on the Projects screen -- projects never load
   > Make sure the API is running on your local machine, and that your **api/.env** and **app/.env** files are configured correctly (see [Setup and first run](#setup-and-first-run) above)
 
+  > Check the URL for both `API_TUNNEL_SUBDOMAIN` and `STA_API_BASE_URL`. `STA_API_BASE_URL` should be prefixed with `https://` but not `API_TUNNEL_SUBDOMAIN`. 
+  -     `STA_API_BASE_URL=https://nadia-bloggs-abc123def456ghi789.loca.lt` 
```

Comment:
Please can you add double quotes `"` around the values on this and the following line e.g. `STA_API_BASE_URL="https://nadia-bloggs-abc123def456ghi789.loca.lt"`  Sometimes it can cause problems in .env files if there aren't quotes around values.

Reply:
Hi Joe i have added in the double quotations